SpatialGDK/Source/SpatialGDK/Public/Utils/SchemaUtils.h

```diff
@@ -129,10 +129,7 @@ inline void AddObjectRefToSchema(Schema_Object* Object, Schema_FieldId Id, const
 	{
 		AddObjectRefToSchema(ObjectRefObject, UNREAL_OBJECT_REF_OUTER_ID, *ObjectRef.Outer);
 	}
-	if (ObjectRef.bUseSingletonClassPath)
-	{
-		Schema_AddBool(ObjectRefObject, UNREAL_OBJECT_REF_USE_SINGLETON_CLASS_PATH_ID, ObjectRef.bUseSingletonClassPath);
-	}
+	Schema_AddBool(ObjectRefObject, UNREAL_OBJECT_REF_USE_CLASS_PATH_TO_LOAD_ID, ObjectRef.bUseClassPathToLoadObject);
```

Comment:
Why change this usage from an optional?

Reply:
It didn't make sense being an optional to me

Reply:
If it wasn't set it was considered false.
I see the schema itself is still an optional.
Done for bandwidth reasons I believe, do we know the tradeoff between an empty optional, and a bool? Same usage?

Reply:
Oh I thought I changed the schema too. My opinion is that having a optional<bool> over just a bool for something infrequently changing just adds more ambiguity than potential bits saved per minute than it's worth? If option<bool> was 100% efficient than bool wouldn't we then always just use bool?

Reply:
Reverted to being optional in schema for network benefit of sending nothing where the value is false. Kept as just a bool in the c++ class.